### ISSUE TITLE
[Bug]   Class "Backpack\CRUD\ViewNamespaces" not found in 5.3.6

### ISSUE THREAD

**Issue by ziming** [Laravel-Backpack/CRUD#4629]
After doing a composer update to 5.3.6 I get this

```bash
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi

   Error 

  Class "Backpack\CRUD\ViewNamespaces" not found

  at vendor/backpack/pro/src/AddonServiceProvider.php:19
     15▕ 
     16▕     public function boot(): void
     17▕     {
     18▕         $this->autoBoot();
  ➜  19▕         ViewNamespaces::addFor('buttons', $this->vendorName.'.'.$this->packageName.'::buttons');
     20▕         ViewNamespaces::addFor('columns', $this->vendorName.'.'.$this->packageName.'::columns');
     21▕         ViewNamespaces::addFor('fields', $this->vendorName.'.'.$this->packageName.'::fields');
     22▕         ViewNamespaces::addFor('filters', $this->vendorName.'.'.$this->packageName.'::filters');
     23▕         ViewNamespaces::addFor('widgets', $this->vendorName.'.'.$this->packageName.'::widgets');      

      +7 vendor frames 
  8   [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(Backpack\Pro\AddonServiceProvider))

      +5 vendor frames 
  14  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

**Comment by pxpm** [comment]
What's your PRO version ? You should be getting `1.2.6`

**Comment by remipou** [comment]
same issue

**Comment by pxpm** [comment]
We are on it. expect the fix in the next coulple minutes! 

Sorry, we somehow messed up the mergin process an some functionality that should be in those commits is missing.

thanks again @ziming 🙏

**Comment by pxpm** [comment]
It's been fixed ... some weird thing happened and we needed to tag 2 versions, 5.2.7 should be good!

Let me know if it's not working for you and we re-open it!

Thank you all for understanding and sorry for the bad experience 🙏 

Cheers

**Comment by tabacitu** [comment]
Thanks so much for bringing this up @ziming & @remipou 🙏 Like Pedro said, a `composer update` should fix it now.

## Postmortem

**TLDR: It was me. It's always me 🤦‍♂️**

Apparently [the 5.3.6 release did NOT include all the stuff in its changelog](https://github.com/Laravel-Backpack/CRUD/compare/5.3.5...5.3.6). Somehow between ReleaseDrafter and Github both writing the release notes, it _looked like_ all those commits were there, but they weren't:
- the release notes were looking at `main`
- the release ITSELF was probably pointing at a `/ref/`, not `main` - I notice Github has been doing this for a few weeks/months; but the ref was always the last one, so the same thing as `main`;

Even though it normally works from `ref` too... most times I manually choose `main`, just to be 100% sure. This time... I guess I did not. And the `ref` github/release-drafter chose was... not what we wanted. I didn't double-check that we're releasing from `main`. I didn't double-check in the demo that everything's still working, after a `composer update`. My bad guys! I'll be more careful from now on, for sure!

Sorry for the frustration 🙏
And thanks again for letting us know (and special thanks to Pedro and his emergency call 😅)

**Comment by crlcu** [comment]
Feels like this issue is still present - https://flareapp.io/share/w5BzNDd5#F93
See the versions I'm currently using:
![image](https://user-images.githubusercontent.com/1661844/187872206-7157a6b7-9ea4-4063-9171-595cd4be3a81.png)

**Comment by ziming** [comment]
add a ^ to the start of ur backpack pro version

**Comment by crlcu** [comment]
This is what I'm getting now:
![image](https://user-images.githubusercontent.com/1661844/187875142-ced29afd-532f-4401-afb7-44025d9fbace.png)

**Comment by pxpm** [comment]
Add your token credentials that you can get from https://backpackforlaravel.com

**Comment by crlcu** [comment]
I have them in the `auth.json`

![image](https://user-images.githubusercontent.com/1661844/187880384-d0dcb104-1347-4d7c-a0d9-f59aae85b596.png)
_instead of **** I have the actual password_

**Comment by pxpm** [comment]
Sorry, i misread, is your Github credentials, not the Backpack ones. 
Did you followed the instructions in the error message? 

Cheers

**Comment by pxpm** [comment]
Do you have the repositories in your `composer.json`  ? 
```php
"repositories": [
        {
            "type": "composer",
            "url": "https://repo.backpackforlaravel.com/"
        }
    ],
```

**Comment by crlcu** [comment]
Right ok,

This explains what's going on :)
<img width="714" alt="image" src="https://user-images.githubusercontent.com/1661844/187895861-5e721005-0447-4bd8-93c3-6addf2761136.png">

How do I fix this issue in this situation then?

**Comment by tabacitu** [comment]
Haha - perfect timing 😅🤦‍♂️ 

Adrian - email me hello@backpackforlarave.com with your account email - I'll gladly add a few days of extra access, so you have this solved. It was my mistake, after all.

**Comment by tabacitu** [comment]
Damn. We didn't think of that @pxpm

**Comment by pxpm** [comment]
Hey @crlcu you should be able to fix it by following the instructions on the message, add `"backpack/pro": "1.0.12"` in your composer.json and run the composer again.

I've just tested the compatibility and I got no issues:
![image](https://user-images.githubusercontent.com/7188159/187901005-3bfc5cd9-1e5a-4a27-88ea-a3ccf889efb4.png)

Let me know if you was able to fix it.

Cheers

**Comment by crlcu** [comment]
Works as expected now,

Cheers.
